src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtTraceActivityInfrastructure.java

```diff
@@ -1,7 +1,16 @@
 package org.prebid.server.proto.openrtb.ext.response;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
+@JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION)
+@JsonSubTypes({
+        @JsonSubTypes.Type(ExtTraceActivityInvocation.class),
+        @JsonSubTypes.Type(ExtTraceActivityInvocationDefaultResult.class),
+        @JsonSubTypes.Type(ExtTraceActivityRule.class),
+        @JsonSubTypes.Type(ExtTraceActivityInvocationResult.class)
+})
```

Comment:
Why do we need this? We already discussed that if someone needs this, they are using the wrong stage. 
Please delete this.